client/src/pages/Courses/editables/TextField.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Box, Text, FormControl, FormLabel, Input, Divider } from "@chakra-ui/core";
+import { colors as c } from "../../../colors";
+
+const TextField = ({ isRequired = false, name, title, value, isEditing, onChange }) => {
+
+    return (
+
+        !isEditing
+            ? (
+                <Box>
+                    <Text color={c.darkBlue} fontWeight="bold">{title[0].toUpperCase() + title.substring(1)}</Text>
```

Comment:
Will title ALWAYS be defined, surely we do a check

Reply:
It should be always defined otherwise there's no point in having the input